src/Sylius/Bundle/ShopBundle/Resources/views/Account/requestPasswordReset.html.twig

```diff
@@ -22,7 +22,9 @@
                 {{ sonata_block_render_event('sylius.shop.request_password_reset_token.form', {'form': form}) }}
 
                 {{ form_row(form._token) }}
-                <button type="submit" class="ui fluid large primary submit button">{{ 'sylius.ui.reset'|trans }}</button>
+                <button type="submit" class="ui fluid large primary submit button" {{ sylius_test_attribute('data-request-password-reset-button') }}>
```

Comment:
I would be for adding `data-` prefix in extension, so you could have `{{ sylius_test_attribute('request-password-reset-button') }}`